rebar.config

```diff
@@ -1,2 +1,17 @@
 {edoc_opts, [{preprocess, true}]}.
 {erl_opts, [debug_info]}.
+{dialyzer, [
+    {warnings, [
```

Comment:
I believe these are the most important non-default options - not count `race_conditions`.